conda_forge_tick/update_upstream_versions.py

```diff
@@ -189,9 +189,11 @@ def update_upstream_versions(gx, sources=(PyPI(), CRAN(), Github(),
                                           RawURL()
                                           )):
     for node, attrs in gx.node.items():
-        attrs['new_version'] = get_latest_version(attrs, sources)
-        logger.info('{} - {} - {}'.format(node,
-                    attrs['version'], attrs['new_version']))
+        print(node, attrs['version'], attrs['new_version'])
```

Comment:
can this go back to the logger?

Reply:
And most likely needs to be below the execution for attrs. (also we should have a logger.warning for the exception)